Sources/FluentKit/Query/Builder/QueryBuilder.swift

```diff
@@ -38,34 +41,35 @@ public final class QueryBuilder<Model>
             self.query.customIDKey = idKey
         }
     }
-
+    
     public func copy() -> QueryBuilder<Model> {
         .init(
             query: self.query,
             database: self.database,
             models: self.models,
             eagerLoaders: self.eagerLoaders,
-            includeDeleted: self.includeDeleted
+            includeDeleted: self.includeDeleted,
+            forceDelete: self.forceDelete
         )
     }
-
+    
     // MARK: Fields
-
+    
```

Comment:
Can you trim whitespace lines? 
![Screen Shot 2020-04-20 at 3 29 26 PM](https://user-images.githubusercontent.com/1342803/79791688-bc9d4300-831b-11ea-8a44-bc9bafed3ac2.png)
(under "while editing")

Reply:
🤯 I had no idea this setting was a thing